server/PythonClient/Client.py

```diff
@@ -0,0 +1,339 @@
+import struct
+import socket
+import time
+import os
+import json
+import abc
+from typing import List, Type, Callable, NewType
+from io import BufferedReader
+import mimetypes
+import math
+import uuid
+
+
+HOST = 'localhost'    # The remote host
+PORT = 9876         # The same port as used by the server
+
+# TOTAL PAYLOAD SIZE
+PAYLOAD_LENGTH = 4
+
+# SIZE OF THE PAYLOAD IDENTIFIER
+PAYLOAD_NAME_LENGTH = 4
+
+# SIZE OF SEGMENTS HEADER
+SEGMENTS_LENGTH = 4
+
+# SIZE OF JSON PAYLOAD
+JSON_LENGTH = 4
+
+ENDIANES = "big"
+
+ENCODING = "UTF-8"
+
+TMP_DIR = "./tmp/"
+
+# Mimics Python socket.sendall signature
+SendFunction = NewType('SendFunction', Callable[[bytes, int], None])
+RecieveFunction = NewType('RecieveFunction', Callable[[bytes, int], None])
```

Comment:
fårstår ikke helt hvorfor dem her er her, kan du forklare

Reply:
Var for å få til typer på egen definert signatur, men tro ikke det virka med Callable